### ISSUE TITLE
Empty file created

### ISSUE THREAD

**Issue by levynir** [ankane/secure-spreadsheet#7]
Running the following command produces an empty excel file when using the attached .csv file.
```
secure-spreadsheet --password 123 < ../tmp/dayzz_leads.csv > ../tmp/dayzz_leads.xlsx
```

I am on MacOS 10.15.7 using `secure-spreadsheet@0.1.2` on `node@12.20.1`
This also happened on an older versions and newer versions of node. 
Encrypting a simple file with just `a,b` works fine.

Input: [dayzz_leads.csv.txt](https://github.com/ankane/secure-spreadsheet/files/5977762/dayzz_leads.csv.txt) (had to add .txt extension since github does not allow csv files).
Output: [dayzz_leads.xlsx](https://github.com/ankane/secure-spreadsheet/files/5977761/dayzz_leads.xlsx)

Thanks

**Comment by ankane** [comment]
Hey @levynir, thanks for reporting! It looks like the error message wasn't being shown. I'll put out a new release shortly.

**Comment by levynir** [comment]
Confirmed fixed on 0.1.3
